### PR TITLE
Fix #430

### LINKED ISSUE #430: Crash when attempting to use build tool after scanning structure with doubleslabs mod blocks
### Prerequisites 

- [x] I am running the latest **alpha** version of Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure that my issue is not covered there.
- [x] I made sure that **this mod compatibility issue is not a duplicate of any existing mod compatibility issue.**

### Context 

- Minecraft Version: 1.16.5
- Structurize Version: 0.13.173-ALPHA-universal and 0.13.210-ALPHA-universal
- Related Mods and their Versions: DoubleSlabs-1.16-3.5.2 and DoubleSlabs-1.16-3.7.1

## Expected behavior

my scanned structure being built

## Actual behavior

Scan tool saves a blueprint of the structure. Build tool crashes on activation when the blueprint file is present in the scans folder.

## Steps to reproduce the issue

1. have the blueprint in the scans folder
2. right click with the build tool to bring up the interface
3. crash

### Logs

- latest.log: https://gist.github.com/codec-d/eebd3700d30bb2bae569b59f657526ab
- crashlog: https://gist.github.com/codec-d/ba4db836320a3ba75fdb0b93005a0b6e

### Notes

I tried removing optifine and that didn't fix the issue. I tried newer releases, no fix. I tried removing double slab mod and rescanning, I was able to place a version of the structure then, with the double slab blocks missing. my original structure only exists in the blueprint file now so I'm just trying to retrieve it in order to use worldedit or mcedit. I can provide the .blueprint file. 
zipped the blueprint to provide it here.
[Mansion_final.zip](https://github.com/ldtteam/Structurize/files/7107231/Mansion_final.zip)
---

## Viewers

* Add a thumbs-up to the mod compatibility issue if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

### PR DESCRIPTION
**Pull request by Nightenom**
Closes #430

Review please

### CODE REVIEW

File: src/main/java/com/ldtteam/structures/client/BlueprintChunk.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
      */
     public BlueprintChunk(final World worldIn, final int x, final int z)
     {
-        super(worldIn, new ChunkPos(x, z), new BiomeContainer(worldIn.registryAccess().registryOrThrow(Registry.BIOME_REGISTRY), new Biome[0]));
+        super(worldIn, new ChunkPos(x, z), new BiomeContainer(null, new Biome[0]));
```
Comment (Raycoms) [inline src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: did you test this, won't this cause potential crashes?

Comment (Raycoms) [reply src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: @Nightenom

Comment (Nightenom) [reply src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: Still crashes, need more investigation on how to properly use our instances (so we don't expose vanilla world for unwanted changes)

Comment (Raycoms) [reply src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: Btw, this should be solved with my changes. as I solved registryAccess() returning null.

Comment (Nightenom) [reply src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: like this is 1.17, so probably no need to care xd, iirc in 1.16 this is not a thing

edit: close if u want

Comment (Raycoms) [reply src/main/java/com/ldtteam/structures/client/BlueprintChunk.java:67]: I'll close the PR and port my fix over later today

**Pull request closed without merging**